testing/vcs/test_script_for_py.py

```diff
@@ -0,0 +1,15 @@
+import vcs
+
+list=["boxfill","meshfill","isofill","textcombined","texttable","vector"]
```

Comment:
Also, you're missing a few:

```
"isoline", "1d", "taylordiagram", "3d_scalar", "3d_dual_scalar", "3d_vector"
``` 

Remove "textcombined" and "texttable"; these should be handled in a second loop, which will do secondary objects (marker, fillarea, colormap, line, textcombined, texttable, textorientation).